### DISCUSSION TITLE
Default public read permission for all files uploaded to s3

### DISCUSSION THREAD

**Discussion by stevemcmillen** [iterate-ch/cyberduck#15741] [Q&A]
I want to set all files I upload to AWS S3 as public read.  I was hoping the following would do it but when I upload files by copy and pasting them from another location on my mac, they do not have public read set.

<img width="496" alt="SCR-20240315-sszf" src="https://github.com/iterate-ch/cyberduck/assets/2750330/9af9e913-01f4-4c04-a7de-fc597aa25264">

How can have all files I upload be public readable without having to edit permissions after upload?

**Comment by dkocher** [comment]
When setting the _Default ACL_ in _Preferences → S3_ as indicated by your screenshot all files uploaded should have public-read ACL set as [documented](https://docs.cyberduck.io/protocols/s3/#default-acls) unless you have bucket owner controls set that disable the user of ACLs [^1].

[^1]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html
